FIPS/pom.xml

```diff
@@ -958,6 +958,46 @@
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
I heard about idea of move duplications in pom and FIPS pom to some kind of parent pom. It is planned for future? Do we have some task?

Reply:
I also heard about that but I can't find a task for it. Anyway - this is definitely out of scope of this PR.

Reply:
I agree - it's out of scope